### DISCUSSION TITLE
本地验证码模块加载失败

### DISCUSSION THREAD

**Discussion by peigenr** [mikumifa/biliTickerBuy#195] [提问专区]
![Uploading B25918BD-12BF-4FCD-963F-28A4C538736F.jpeg…]()
打开程序后遇见了这个问题，请问会影响后续的抢票和过验证码吗，那应该怎么解决呢

**Comment by mikumifa** [comment]
图片没更新好

**Comment by peigenr** [comment]
不知道为什么显示不出来，手打吧（
错误信息：DLL load failed while importing bili_ticket_gt_python：找不到指定的模块

**Comment by mikumifa** [comment]
还是没有，你要等一会

**Comment by mikumifa** [comment]
系统太老了，只能手动抢票了

**Comment by peigenr** [comment]
系统版本是windows10 LTSC 2019
